docs/user-guide/side-bar.md

```diff
@@ -91,7 +95,7 @@ An example of search with `Aeronautical` coordinates as follows:
 
 ### Identify Tool with more than one layer
 
-On a map it is possible to have several overlapping layers. With the *Identify* tool the user can retrieve information on one or more overlapping layers at the same time.
```

Comment:
In a map it is possible to have several overlapping layers. With the *Identify* tool the user can retrieve information on one or more overlapping layers at the same time in a certain point.